auth.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const crypto = require('crypto');
+
+const usernameExists = require('./queries/usernameExists');
+const userIsAuthorized = require('./queries/userIsAuthorized');
+const fetchUser = require('./queries/fetchUser');
+const setToken = require('./services/setToken');
+const addUser = require('./services/addUser');
+
+const generateRandomValue = (bytes = 32) => {
```

Comment:
Now you should split this file into all the services it comprises and require those only in the controllers/services you need them